cpp/velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -375,6 +383,9 @@ class SubstraitToVeloxPlanConverter {
 
     bool nullAllowed_ = false;
     bool isNull_ = false;
+    bool forbidsNullSet_ = false;
+    bool isNullSet_ = false;
+    bool existIsNullAndIsNotNull_ = false;
```

Comment:
Can we just depend on the check for `forbidsNullSet_ && isNullSet_` instead of introducing a new flag?

Reply:
I'll address this later soon.